models/chef/Chef.validation.js

```diff
@@ -0,0 +1,16 @@
+import Joi from 'joi';
+import { addressValidationSchema } from '../order/order.validation';
+import { accountStatus } from '../../constants/chefEnums';
+
+const ChefValidationSchema = Joi.object({
+  userId: Joi.required(),
+  chefImage: Joi.string().required(),
+  address: addressValidationSchema.required(),
+  certificate: Joi.string().required(),
+  accountStatus: Joi.string()
+    .valid(...accountStatus)
+    .required(),
+  isAvailable: Joi.boolean().default(true),
```

Comment:
why did you use `default` here, but did not use it for `accountStatus`? consistency is always better, either use default for both or don't use default and rely on model